CONTRIBUTING.md

```diff
@@ -1,31 +1,22 @@
 ## Contributing
 
-This package is managed via npm so you will need to create a user and be added to the teamsnap org in order to publish updates.
-
-Once your user is created and added to the org this guide details the steps to publish an update to npm.
-
-<https://docs.npmjs.com/getting-started/publishing-npm-packages>
-
 #### Steps for publishing an update
 
-* rebase and get  your branch ahead of master
+* rebase and get your branch ahead of master
 * then from your working branch
 * run `yarn npm audit` to ensure there are no vulnerabilities with level high or critical.
 * run `yarn run build` to ensure the package builds with no errors
 * As necessary, update packages to resolve npm audit failures for level high or critical vulnerabilities.
-* Update the package.json version number `npm version [major|minor|patch] -m "commit message"`
-* Update CHANGELOG.md with new npm version and release notes
-* Commit CHANGELOG
+* Use conventional commits to commit changes to the codebase (https://www.conventionalcommits.org/en/v1.0.0/)
 * `git push origin branch-name`
-* merge branch into master from git
-* `git checkout master`
-* `git pull origin master`
-* `npm login --scope=@teamsnap`
-* `npm publish`
+* merge branch into v4 from github
 * To update TeamSnap CSS assets on CDN, run `jarvis deploy teamsnap_ui/master` in #chatops
 * As necessary, update CSS version number in repositories that reference it:
   - Classic (for NJW): https://github.com/teamsnap/classic/blob/master/app/views/_header_head_tag.html.erb#L33
 
+#### What gets published?
+We use semantic release and conventional commits to determine what triggers a publish. You no longer need to publish v4, as our CI/CD pipeline will take care of that for you, however, you do have to follow the conventional commit message spec to trigger a build. Commitizen is a fantastic tool to help you follow the convention commit spec and we strongly recommend using that.
```

Comment:
Do we have documentation or examples on how to setup and use `commitizen` for engineers?

Reply:
We do not. I've updated the readme to include links to commitizen, semantic release, and conventional commits. We can probably find a good home for setup instructions too, if needed.